packages/react-core/src/demos/examples/Page/PageContextSelector.tsx

```diff
@@ -325,13 +325,13 @@ export const PageStickySectionBreadcrumb: React.FunctionComponent = () => {
       mainContainerId={mainContainerId}
       isBreadcrumbWidthLimited
     >
-      <PageSection isWidthLimited>
+      <PageSection isWidthLimited aria-label="Main title">
```

Comment:
Similar as th eother PR, let's point to the Content component's id in an aria-labelledby instead